src/piggui.rs

```diff
@@ -314,10 +309,25 @@ fn get_hardware_target(matches: &ArgMatches) -> HardwareTarget {
     target
 }
 
-#[cfg(not(target_arch = "wasm32"))]
 /// Parse the command line arguments using clap
 fn get_matches() -> ArgMatches {
-    let app = clap::Command::new(env!("CARGO_BIN_NAME")).version(env!("CARGO_PKG_VERSION"));
+    let app = clap::Command::new(env!("CARGO_BIN_NAME")).override_help(
```

Comment:
You shouldn't need to do this.

Clap has methods to do this, and that will be updated automatically when we add option susing clap